eng/Publishing.props

```diff
@@ -1,5 +1,9 @@
 <Project>
   <PropertyGroup>
+    <PropertyGroup>
```

Comment:
Typo that will break official builds as soon as you check this in

Reply:
Ya. Looks like the official test build didn't have this typo.

Reply:
Good catch! I fixed that.